index.test-d.ts

```diff
@@ -0,0 +1,11 @@
+import { expectType } from 'tsd-check';
+import readPkg from '.';
+import { IPackage } from '.';
+
+(async () => {
```

Comment:
You don't have to wrap it in an async function. `tsd-check` supports top-level await.

Reply:
I saw that, but I wrapped it anyways because my editor (VS Code) complains about the top-level await, even if `tsd-check` supports them.

Should I change this anyways?

Reply:
> Should I change this anyways?

Yes